components/CatModal/CatModalView.tsx

```diff
@@ -86,16 +86,16 @@ export default function CatModalView({
       observations: '',
       mediaLinks: [],
       ...(cat ?? {}),
-      volunteerName: (cat as SterilizedCat)?.volunteerName || '',
-      hospitalizationDate: new Date((cat as SterilizedCat)?.hospitalizationDate || Date.now()),
-      releaseDate: new Date((cat as SterilizedCat)?.releaseDate || Date.now()),
-      sterilizedStatus: (cat as SterilizedCat)?.releaseDate ? 'sterilized' : 'unsterilized',
+      volunteerName: (cat as SterilizedCat)?.volunteerName ?? '',
+      hospitalizationDate: (cat as SterilizedCat)?.hospitalizationDate ? new Date((cat as SterilizedCat)?.hospitalizationDate) : undefined,
+      releaseDate: (cat as SterilizedCat)?.releaseDate ? new Date((cat as SterilizedCat)?.releaseDate) : undefined,
+      sterilizedStatus: (cat as SterilizedCat)?.volunteerName !== undefined ? 'sterilized' : 'unsterilized',
     },
     validationRules: {
       observations: (value) => value.length > 0 || value.length === 0 && existingImages.length > 0,
-      hospitalizationDate: (value, other) => value ? value! <= other?.releaseDate! || value! <= new Date() : true,
-      releaseDate: (value, other) => value ? value! <= new Date() && other?.hospitalizationDate! <= value! : true,
-      volunteerName: (value, other) => other?.sterilizedStatus === 'sterilized' ? (nameRegex.test(value!)) : true
+      hospitalizationDate: (value, other) => value ? value! <= other?.releaseDate! || value! <= new Date() : other?.sterilizedStatus === 'unsterilized' || (modal.type === 'ADD_CAT'),
+      releaseDate: (value, other) => value ? value! <= new Date() && other?.hospitalizationDate! <= value! : other?.sterilizedStatus === 'unsterilized' || (modal.type === 'ADD_CAT'),
```

Comment:
Please add a few parenthesis to the above conditions, they are difficult to follow as they are